framework/src/executioners/Transient.C

```diff
@@ -474,6 +474,8 @@ Transient::endStep(Real input_time)
 
   _picard_converged=false;
 
+  bool ts_output =_time_stepper->enableOutput();
```

Comment:
Sorry to nit-pick but since I see two minor issues in such a short piece of code, I'll comment. First, our code standard says spacing around binary operators so you need a space after the '='. Second. I don't see why you need this temporary at all here. You don't use it until later and there's no loop or anything to optimize. Just retrieve it right when you need it, or at least put it next to the place where it's used.